doc/macsec/MACsec_hld.md

```diff
@@ -762,6 +779,50 @@ All boxes with black edge are components of virtual SAI and all boxes with purpl
 
 ***MACsec egress sc will be automatically created/delete when the MACsec port is created/deleted***
 
+#### 3.4.6 PFC with MACsec
+
+To leverage the capability of ACL to bypass or drop the PFC and pause frames like what EAPOL did.
```

Comment:
should add more context, in general how is pfc handled in such system, is gearbox doing pfc process, or is gearbox doing the pfc processing.